src/main/java/io/xol/enklume/MinecraftRegion.java

```diff
@@ -103,12 +88,4 @@ private MinecraftChunk getChunkInternal(int x, int z) {
         }
         return new MinecraftChunk(x, z);
     }
-
-    public void close() {
```

Comment:
Not closing files isn't amazing, then again maybe there is already a finalizer taking care of this ? I don't know the fine print here